Kafka bug on master: assignment to entry in nil map

With current version of master, running : 
`--input-raw :3000 --input-raw-track-response --output-kafka-host mykafka:port --output-kafka-topic mytopic --output-kafka-json-format --http-set-header "GoReplay-Origin: mytest"`

Leads to : 
```
panic: assignment to entry in nil map

goroutine 98 [running]:
main.(*KafkaOutput).PluginWrite(0xc000613080, 0xc0005222a0, 0x0, 0x0, 0x0)
        /xxxx/output_kafka.go:75 +0x221
main.CopyMulty(0x7f025ce4e538, 0xc0002f8000, 0xc00060e150, 0x1, 0x1, 0x0, 0x0)
        /xxxx/emitter.go:155 +0x1516
main.(*Emitter).Start.func2(0xc0006130e0, 0xc0001048c0, 0x7f025ce4e538, 0xc0002f8000)
        /xxxx/emitter.go:53 +0xe3
created by main.(*Emitter).Start
        /xxxx/emitter.go:51 +0x51c
```

Explanation : hmap header is not initialized in output_kafka.go:73

Seems to be a bug introduce by this commit : 
https://github.com/buger/goreplay/commit/5d8ca525a41057ef1934d0377ad97b7bedca0c32#diff-016d74760d146cc95a2bd4cd10054c2ed3e965ad8432dbdcc14744efa42b5a6fL76

`headers := make(map[string]string)`
becomes
`var header map[string]string` **without make**

--- a/output_kafka.go
+++ b/output_kafka.go
@@ -70,7 +70,7 @@ func (o *KafkaOutput) PluginWrite(msg *Message) (n int, err error) {
 		message = sarama.StringEncoder(byteutils.SliceToString(msg.Meta) + byteutils.SliceToString(msg.Data))
 	} else {
 		mimeHeader := proto.ParseHeaders(msg.Data)
-		var header map[string]string
+		header := make(map[string]string)
 		for k, v := range mimeHeader {
 			header[k] = strings.Join(v, ", ")
 		}

